gatsby-config.js

```diff
@@ -95,6 +97,68 @@ module.exports = {
     'gatsby-transformer-json',
     'gatsby-transformer-sharp',
     'gatsby-plugin-sharp',
+    {
+      resolve: `gatsby-plugin-feed`,
+      options: {
+        query: `
+          {
+            site {
+              siteMetadata {
+                title
+                description
+                siteUrl
+                site_url: siteUrl
+              }
+            }
+          }
+        `,
+        feeds: [
+          {
+            serialize: ({ query: { site, allMarkdownRemark } }) => {
+              return allMarkdownRemark.edges
+                .filter((edge) => !edge.node.fields.slug.includes('/pages/'))
+                .map((edge) => {
+                  return Object.assign({}, edge.node.frontmatter, {
+                    description: edge.node.excerpt,
+                    date: edge.node.frontmatter.date,
+                    url: site.siteMetadata.siteUrl + edge.node.frontmatter.path,
+                    guid:
+                      site.siteMetadata.siteUrl + edge.node.frontmatter.path,
+                    custom_elements: [{ 'content:encoded': edge.node.html }],
+                  });
+                });
+            },
+            query: `
+              {
+                allMarkdownRemark(
+                  sort: { order: DESC, fields: [frontmatter___date] },
+                ) {
+                  edges {
+                    node {
+                      excerpt
+                      html
+                      fields { slug }
```

Comment:
I guess you followed https://www.gatsbyjs.com/docs/how-to/adding-common-features/adding-an-rss-feed/
which recommends adding a slug to the markdown field to query for it the markdown files. I wonder if this as any implications but it works. So it's fine.